### PR TITLE
Add basic Jax support to the datalaoder

### PR DESCRIPTION
**Pull request by benfred**

### CODE REVIEW

File: merlin/loader/loader_base.py [hunk 1]
```diff
@@ -314,7 +316,7 @@ def make_tensors(self, gdf, use_nnz=False):
         """
         split_idx = self._get_segment_lengths(len(gdf))
         # map from big chunk to framework-specific tensors
-        chunks = self._create_tensors(gdf)
+        chunks, names = self._create_tensors(gdf)
```
[the remark this answers is not in the document]
Comment (benfred) [reply merlin/loader/loader_base.py:319]: I think this fix is also needed for https://github.com/NVIDIA-Merlin/NVTabular/pull/1694 to work

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by benfred** [comment]
This seems to be failing in jenkins because of ` horovod does not find an installed MPI.\n\nChoose one of:\n1. Install Open MPI 4.0.0+ or IBM Spectrum MPI or MPICH and re-install Horovod` . 

I set pytest up to skip this test in this config here https://github.com/NVIDIA-Merlin/dataloader/pull/38/commits/7abee1919d09baacbec204d1f9e7f992d518a1f7

**Review by jperez999 — APPROVED** [review]

**Pull request merged**